src/templates/components/topTasks/index.test.tsx

```diff
@@ -0,0 +1,196 @@
+import { TopTasks, _topTaskLovellComp } from './index'
+import { render } from '@testing-library/react'
+
+const lovellAdministration = { entityId: 1039 }
+
+describe('TopTasks', () => {
+  it('should render the normal links', () => {
+    const { container } = render(
+      <TopTasks path="/test-nav-path" vamcEhrSystem="vista" />
+    )
+    expect(
+      container.querySelector('va-link-action[text="Make an appointment"]')
+    ).toBeInTheDocument()
+    expect(
+      container.querySelector('va-link-action[text="Register for care"]')
+    ).toBeInTheDocument()
+    expect(
+      container.querySelector(
+        'va-link-action[text="Learn about pharmacy services"]'
+      )
+    ).toBeInTheDocument()
+  })
+
+  it('should render the MHS link', () => {
+    const { container } = render(
+      <TopTasks
+        path="/test-nav-path"
+        vamcEhrSystem="cerner"
+        administration={{ entityId: 1039 }}
+      />
+    )
+    expect(
+      container.querySelector(
+        'va-link-action[text="MHS Genesis Patient Portal"]'
+      )
+    ).toBeInTheDocument()
+  })
+
+  it('should render the MHS link from the office', () => {
+    const { container } = render(
+      <TopTasks
+        path="/test-nav-path"
+        /* @ts-expect-error Shouldn't happen, but just in case... */
+        vamcEhrSystem=""
+        office={{ vamcEhrSystem: 'cerner' }}
+        administration={{ entityId: 1039 }}
+      />
+    )
+    expect(
+      container.querySelector(
+        'va-link-action[text="MHS Genesis Patient Portal"]'
+      )
+    ).toBeInTheDocument()
+  })
+
+  it('should render the MHS link from the regonPage', () => {
+    const { container } = render(
+      <TopTasks
+        path="/test-nav-path"
+        /* @ts-expect-error Shouldn't happen, but just in case... */
+        vamcEhrSystem=""
+        /* @ts-expect-error Shouldn't happen, but just in case... */
+        office={{ vamcEhrSystem: '' }}
+        regionPage={{ vamcEhrSystem: 'cerner' }}
+        administration={{ entityId: 1039 }}
+      />
+    )
+    expect(
+      container.querySelector(
+        'va-link-action[text="MHS Genesis Patient Portal"]'
+      )
+    ).toBeInTheDocument()
+  })
+
+  it('should handle no slash in the path', () => {
+    const { container } = render(
+      <TopTasks path="test-nav-path" vamcEhrSystem="vista" />
+    )
+    expect(
+      container.querySelector(
+        'va-link-action[href="/test-nav-path/register-for-care"]'
+      )
+    ).toBeInTheDocument()
+  })
+})
+
+describe('topTaskLovellComp', () => {
```

Comment:
This bit is intended to test all the important bits of the "render the right link" function, `topTaskLovellComp`.

Because there's so much going on here, there are technically _waaaaaaaaay_ more combinations we _could_ test, but that was _super_ cumbersome. I tried going that route first. It was no good.

Instead, I went with the "show me the happy-path MHS link" first, then tested changes to that one at a time. I think it's pretty comprehensive without being exhaustive, but let me know if I missed something!